erc20/IERC20Extended.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
+interface IERC20Extended {
```

Comment:
what's the diff between this and the one at the top of the PR?

Reply:
A mistake